crates/ndc-sendgrid/src/mutation.rs

```diff
@@ -47,166 +45,42 @@ async fn process_operation(
                 "Unknown procedure: {unknown_procedure}"
             ))),
         },
-        MutationOperation::Insert { .. } => Err(MutationError::UnsupportedOperation(String::from(
-            "Insert mutations currently not supported",
-        ))),
-        MutationOperation::Update { .. } => Err(MutationError::UnsupportedOperation(String::from(
-            "Update mutations currently not supported",
-        ))),
-        MutationOperation::Delete { .. } => Err(MutationError::UnsupportedOperation(String::from(
-            "Delete mutations currently not supported",
-        ))),
-    }
-}
-
-fn complicate_request(simple_request: SimpleSendMailRequest) -> sendgrid_api::SendMailRequest {
```

Comment:
Aww I loved that name 💔

Reply:
SendGrid's API surface is definitely not simple. :(